src/Framework.UnitTests/ProjectFinishedEventArgs_Tests.cs

```diff
@@ -35,6 +45,58 @@ public void EventArgsCtors()
             projectFinishedEvent = new ProjectFinishedEventArgs(null, null, null, true, DateTime.Now);
         }
 
+        /// <summary>
+        /// Roundtrip serialization tests for <see cref="ProfilerResult"/>
+        /// </summary>
+        [MemberData(nameof(GetProfilerResults))]
+        [Theory]
+        public void ProfilerResultRoundTrip(ProfilerResult profilerResult)
```

Comment:
Shouldn't this test go in a new ProjectEvaluationFinishedEventArgs_Tests class instead?

Reply:
Yeap, good catch